common-lib/src/main/scala/com/gu/mediaservice/model/ImageMetadata.scala

```diff
@@ -6,6 +6,8 @@ import play.api.libs.functional.syntax._
 import play.api.libs.json._
 
 case class ImageMetadata(
+/* these are standard metadata fields that exist in multiple schemas,
+most canonical being https://www.iptc.org/std/photometadata/specification/IPTC-PhotoMetadata */
```

Comment:
Traditionally this comment would be before the start of the class if referring to the entire class and above a field if only referring to a field. Thus this looks like it just applies to `dateTaken` which is obviously not what you mean....

Reply:
Tradition is correct. Here.